### ISSUE TITLE
Ability to execute Python code block in a PowerShell Notebook 

### ISSUE THREAD

**Issue by MaximoTrinidad** [dotnet/interactive#566]
**Is your feature request related to a problem? Please describe.**
I haven't been able to locate the SQL Kernel so I can use to create SQL Notebook. I've seen it Azure Data Studio but can't find it for .NET Interactive.

**Describe the solution you'd like**
I know that in a C# Notebook you can use the "#!" to execute PowerShell command. So, it would be nice to have it also for Python or any other Kernels. 

The scenario I'm looking for, I want to create a connection using Python code to then execute the SQL code. Of course, required python all modules are previously installed.

Is this possible? 

It would be Great!
:)

**Describe alternatives you've considered**

**Comment by dbrattli** [comment]
Yes it would be great to have multi language notebooks with Python. In particular F# and Python. I got surprised it didn't work. You also need Python with notebooks to make the usable.
